cfme/tests/infrastructure/test_publish_vm_to_template.py

```diff
@@ -3,14 +3,15 @@
 
 from cfme.infrastructure.provider import InfraProvider
 from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
 from cfme.markers.env_markers.provider import ONE_PER_TYPE
-from cfme.utils.blockers import BZ
 from cfme.utils.generators import random_vm_name
 from cfme.utils.log import logger
 from cfme.utils.virtual_machines import deploy_template
 
 pytestmark = [
-    pytest.mark.provider([InfraProvider])
+    pytest.mark.provider([InfraProvider]),
```

Comment:
This is not the proper way to filter the providers, you only want one mark call here.

Should look like:
```
pytestmark = [pytest.mark.provider([InfraProvider], required_fields=[['templates', 'small_template']])]
```